apps/mobile/src/modules/documents/documents.services.ts

```diff
@@ -72,3 +74,35 @@ export async function fetchOrganizationDocuments({
     throw error;
   }
 }
+
+export async function fetchDocumentFile({
+  document,
+  organizationId,
+  baseUrl,
+  authClient,
+}: {
+  document: Document;
+  organizationId: string;
+  baseUrl: string;
+  authClient: AuthClient;
+}) {
+  const cookies = authClient.getCookie();
+  const uri = `${baseUrl}/api/organizations/${organizationId}/documents/${document.id}/file`;
+  const headers = {
+    'Cookie': cookies,
+    'Content-Type': 'application/json',
+  };
+  // Use documentDirectory for better app compatibility
+  const fileUri = `${FileSystem.cacheDirectory}${document.name}`;
```

Comment:
Nice, seems better with cache directory

[Nit] Comment

```suggestion
  // Use cacheDirectory for better app compatibility
  const fileUri = `${FileSystem.cacheDirectory}${document.name}`;
```